http/1.5/modules/ROOT/pages/http-consume-web-svc-task.adoc

```diff
@@ -1,33 +1,96 @@
-= To Consume a REST Service (Studio)
+= Consume a REST Service Example - Mule 4
 :page-aliases: connectors::http/http-consume-web-svc-task.adoc
 
-You can design an app to consume a REST service by setting up HTTP Listener and Request operations. In this procedure, you first consume a REST API service using primarily default HTTP configurations. Next, you test the HTTP request. You edit the configuration to make a secure HTTPS request. Finally, you test the HTTPS request.
+In the following example you configure Anypoint Connector for HTTP (HTTP Connector) to consume a REST service by setting up the HTTP *Listener* and *Request* operations.
```

Comment:
Add a comma after "example."

...HTTP *Listener* source and *Request* operation.